core/controllers/blog_admin.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Controllers for the blog admin page"""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import logging
+
+from core.controllers import acl_decorators
+from core.controllers import base
+from core.domain import blog_services
+from core.domain import config_domain
+from core.domain import config_services
+from core.domain import role_services
+from core.domain import user_services
+import feconf
+import python_utils
+
+BLOG_POST_EDITOR = feconf.ROLE_ID_BLOG_POST_EDITOR
+BLOG_ADMIN = feconf.ROLE_ID_BLOG_ADMIN
+
+
+class BlogAdminHandler(base.BaseHandler):
```

Comment:
Instead of using 'config tab' in admin page, we will be having a separate page called - 'Blog Admin' page. Blog Admin will be responsible for assigning and removing 'Blog Post Editors', and setting the list of default tags for blogs. Thus new controllers for handling data on 'blog admin' page. Also we don't want admin to be able to set list of tags or update blog editors.

Reply:
Ah ok, makes sense.